pages/product.html

```diff
@@ -0,0 +1,410 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="../dist/output.css" />
+    <title>Product</title>
+  </head>
+  <body>
+    <section class="navbar">
+      <header class="navbar__header">
+        <img class="navbar__image" src="../assets/images/Vector-logo.png" />
+
+        <ul class="navbar__navbar">
+          <li class="navbar__item"><a href="../index.html">Home</a></li>
+          <li class="navbar__item">
+            <a href="./product.html">Product</a>
+          </li>
+          <li class="navbar__item">
+            <a href="./pricing.html">Pricing</a>
+          </li>
+          <li class="navbar__item">
+            <a href="./aboutUs.html">About Us</a>
+          </li>
+          <li class="navbar__item">
+            <a href="./blog.html">Blog</a>
+          </li>
+          <li class="navbar__item">
+            <a href="./contactUs.html">Contact</a>
+          </li>
+          <li class="navbar__btn">Free trial</li>
+        </ul>
+      </header>
+    </section>
+
+    <section class="product">
+      <img
+        src="../assets/images/product-gradient.png"
+        alt=""
+        class="product__gradient"
+      />
+      <div class="product__banner">
+        <p class="product__title">Grow your Sales and Services</p>
+        <p class="product__description">
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+          eiusmod <br />tempor incididunt ut labore et dolore magna aliqua..
+        </p>
+
+        <div class="product__btns">
+          <button class="product__btnactive">Get Free trial</button>
+
+          <button class="product__btn">Get Free trial</button>
+        </div>
+      </div>
+
+      <img
+        src="../assets/images/Dashboard image.png"
+        class="product__image"
+        alt=""
+      />
+
+      <div class="product__logos">
+        <img src="../assets/images/Logo 1.png" class="product__logo" alt="" />
+        <img src="../assets/images/Logo 2.png" class="product__logo" alt="" />
+        <img src="../assets/images/Logo 3.png" class="product__logo" alt="" />
+        <img src="../assets/images/Logo 4.png" class="product__logo" alt="" />
+        <img src="../assets/images/Logo 5.png" class="product__logo" alt="" />
+      </div>
+    </section>
+
+    <section class="product__body">
+      <h1 class="product__body-title">
+        Get the best out of your company <br />
+        with our service
+      </h1>
+      <img
+        src="../assets/images/gradient-banner.png"
+        alt=""
+        class="product__gradient"
+      />
+
+      <div class="product-cards">
```

Comment:
## prohibition of nested block
`product-cards` block inside the block is not a best practice